proto/claim/query.proto

```diff
@@ -15,7 +18,31 @@ service Query {
   rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
     option (google.api.http).get = "/tendermint/spn/claim/params";
   }
-  // this line is used by starport scaffolding # 2
+  // Queries a ClaimRecord by index.
+	rpc ClaimRecord(QueryGetClaimRecordRequest) returns (QueryGetClaimRecordResponse) {
```

Comment:
Seems Ignite CLI doesn't format well proto files. Can we run format in the proto files?